### DISCUSSION TITLE
New Progress Bar in qBitTorrent theme

### DISCUSSION THREAD

**Discussion by witalihirsch** [witalihirsch/qBitTorrent-fluent-theme#15] [Polls]
Do you think the new Progress Bar is better than the old one?
If someone is interested in my opinion, then I like the new one better, because it looks fresher and matches the new windows design

**Comment by carlosvsilva** [comment]
I think the static GUI elements are way oversized relative to the dynamic content itself. I would "zoom out" a fair bit :) 
Other than that, congratulations on the amazing theme!
